modules/currency-l0/src/main/scala/org/tessellation/currency/l0/snapshot/programs/Rollback.scala

```diff
@@ -37,7 +37,7 @@ object Rollback {
     nodeId: PeerId,
     identifier: Address,
     globalL0Service: GlobalL0Service[F],
-    lastSignedBinaryHashStorage: LastSignedBinaryHashStorage[F],
+    lastSignedBinaryHashStorage: LastBinaryHashStorage[F],
```

Comment:
is it `lastSignedBinaryHashStorage` (like the variable name) or `lastBinaryHashStorage` (like the type)?

Reply:
It follows change of how hash is calculated now. It taken directly from binary, not signed binary.

Reply:
yes, so you should change the name of the variable. it's a mismatch now